data-weave-plugin/src/main/java/org/mule/tooling/lang/dw/util/VirtualFileSystemUtils.java

```diff
@@ -126,7 +127,7 @@ private static VirtualFile resolve(Project project, NameIdentifier name, GlobalS
     public static NameIdentifier getNameIdentifierWithRelative(VirtualFile vfs, VirtualFile contentRootForFile) {
         final String relPath = VfsUtil.getRelativePath(vfs, contentRootForFile);
         if (relPath != null) {
-            return NameIdentifierHelper.fromWeaveFilePath(relPath);
+            return NameIdentifierHelper.fromWeaveFilePath(FileSystems.getDefault().getSeparator().equals("\\") ? relPath.replace("/", "\\") : relPath);
```

Comment:
```suggestion
            return NameIdentifierHelper.fromWeaveFilePath(relPath, FileSystems.getDefault().getSeparator());
```

Reply:
This is much simpler and should have the same outcome

Reply:
Indeed, a new fix pushed.  Thanks